### PR TITLE
use JSX through babel-register

### PR DESCRIPTION
**Pull request by jarvys**
All pull requests should go to `master`, even the documentation is in `master`
under `./docs/`.

### CODE REVIEW

File: app/package.json [hunk 1]
```diff
@@ -35,5 +36,11 @@
       "MAS": "3rd Party Mac Developer Application: Company Name (APPIDENTITY)",
       "MASInstaller": "3rd Party Mac Developer Installer: Company Name (APPIDENTITY)"
     }
+  },
+  "devDependencies": {
+    "babel": "^6.5.2",
+    "babel-preset-es2015": "^6.18.0",
+    "babel-preset-react": "^6.16.0",
+    "babelify": "^7.3.0"
```
Comment (bayleedev) [inline app/package.json]: should these be non-dev dependencies if they are ran on the client?

File: app/components/search.js [hunk 2]
```diff
@@ -39,14 +39,25 @@ const Search = React.createClass({
...
+    )
+
+    // return React.createElement('input', {
+    //   title: 'Search Zazu',
+    //   className: 'mousetrap',
+    //   ref: this.setReference,
+    //   type: 'text',
+    //   onChange: this.handleQueryChange,
+    //   value: value,
+    // })
```
Comment (bayleedev) [inline app/components/search.js]: can you remove these comments?

File: app/debug.html [hunk 3]
```diff
@@ -4,6 +4,8 @@
     <meta charset="utf-8">
     <title>Zazu</title>
     <script>
+      require('babel-register');
```
Comment (bayleedev) [inline app/debug.html]: semicolon

**Review by bayleedev — CHANGES_REQUESTED** [review]

**Review by afaur — APPROVED** [review]
👍

**Review by bayleedev — APPROVED** [review]

**Pull request merged**